src/test/spec/unified_runner/test_runner.rs

```diff
@@ -59,7 +59,12 @@ impl TestRunner {
                 TestFileEntity::Client(client) => {
                     let id = client.id.clone();
                     let observe_events = client.observe_events.clone();
-                    let ignore_command_names = client.ignore_command_monitoring_events.clone();
+                    let mut ignore_command_names = client.ignore_command_monitoring_events.clone();
+                    if !client.observe_sensitive_commands.unwrap_or(false) {
```

Comment:
I found the double negation here a little bit difficult to read. I'd be in favor of using `match` here, either leaving the `Some(true)` branch empty or assigning `ignore_command_names` as follows:
```suggestion
                    let ignore_command_names = match client.observe_sensitive_commands {
                        Some(true) => client.ignore_command_monitoring_events.clone(),
                        _ => // same as current logic
```
OR
```suggestion
                   match client.observe_sensitive_commands {
                        Some(true) => {}
                        _ => //same as above
```

Reply:
Done - went with the second.

Reply:
Another option here would be to make the conditional `if client.observe_sensitive_events != Some(true)` -- I think that's a little more readable than the original and avoids the verbosity of a `match` with an empty case.